control_panel_api/tests/__init__.py

```diff
@@ -1,3 +1,5 @@
+from unittest.case import TestCase
```

Comment:
Actually I don't need this in this module. I originally put the `IAMPolicyTestCase` class in this file but then I decided it was better to have it in a separate file. Removing this.